pkgs/by-name/be/beeper-bridge-manager/package.nix

```
@@ -0,0 +1,26 @@
+{ lib
+, buildGoModule
+, fetchFromGitHub
+}:
+
+buildGoModule rec {
+  pname = "bbctl";
+  version = "0.12.0";
+
+  src = fetchFromGitHub {
+    owner = "beeper";
+    repo = "bridge-manager";
+    rev = "v${version}";
+    hash = "sha256-xaBLI5Y7PxHbmlwD72AKNrgnz3D+3WVhb2GJr5cmyfs=";
+  };
+
+  vendorHash = "sha256-VnqihTEGfrLxRfuscrWWBbhZ/tr8BhVnCd+FKblW5gI=";
+
+  meta = with lib; {
+    description = "Tool for running self-hosted bridges with the Beeper Matrix server. ";
+    homepage = "https://github.com/beeper/bridge-manager";
+    license = licenses.asl20;
+    maintainers = with maintainers; [ heywoodlh ];
```

According to #292468 `with` is discouraged now:

```suggestion
  meta = {
    description = "Tool for running self-hosted bridges with the Beeper Matrix server. ";
    homepage = "https://github.com/beeper/bridge-manager";
    license = lib.licenses.asl20;
    maintainers = [ lib.maintainers.heywoodlh ];
```

---

Fixed on amended commit, thanks for the feedback!